src/hooks/api/getters/useSegments/useSegments.ts

```diff
@@ -18,7 +18,10 @@ export const useSegments = (strategyId?: string): IUseSegmentsOutput => {
 
     const { data, error, mutate } = useSWR(
         [strategyId, uiConfig.flags],
-        fetchSegments
+        fetchSegments,
+        {
+            refreshInterval: 15 * 1000,
```

Comment:
Curious: Why was this needed?

Reply:
Not strictly needed. It updates this list of other people add/remove items. Might be nice to add this as default for SWR.